core/modules/user/user.module

```diff
@@ -1630,6 +1636,19 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the user page tabs setting.
+ */
+function user_login_page_settings() {
```

Comment:
Because this only provides a single setting as a string, we should maybe change this function name (and the settings key). In the issue I suggested calling the setting "user_login_appearance". So this might become:
```suggestion
function user_login_appearance_setting() {
```

Reply:
I already noted in a previous PR, which also adds that function, that I think, it provides no value to create a function for one single config item to statically cache an item from 'system.core', which is already statically cached. This function could simply get dropped.

Reply:
Done